app/assets/stylesheets/bootstrap-headerfix.scss

```diff
@@ -41,11 +41,12 @@ header {
         font-weight: bold;
       }
     }
-    .notification_element {
+
+    .media.stream_element{
       font-size: 13px;
-      .timeago {
-        border: medium none;
-        cursor: text;
+      img.avatar {
+        width: 35px;
+        height: 35px;
```

Comment:
I put it here because Boostrap's rule aren't overriden otherwise.

![capture d ecran 2015-03-10 a 22 02 54](https://cloud.githubusercontent.com/assets/3429231/6585159/70fb3400-c770-11e4-9cc4-a07de7259713.png)